addons/survey/tests/test_survey_invite.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
I don't think those change should be in commit "[REF] hr_recruitment_survey: adapt application to survey ACLs update". (Neither changing the field type of `deadline` on `survey.survey`). ^^

Reply:
Moved to the right commit. Thanks :) .